.github/workflows/release.yml

```diff
@@ -53,12 +53,37 @@ jobs:
     runs-on: windows-latest
     needs: upload
     steps:
+      - run: echo Calling setup-node with node-version ${{ vars.NODE_VERSION }}
       - uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # v4.1.2
       - uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
         with:
-          node-version: 18
+          node-version: ${{ vars.NODE_VERSION }}
           registry-url: https://registry.npmjs.org/
       - run: |
              lerna publish from-package --no-private --yes
         env:
-          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        working-directory: ${{ github.workspace }}/src/shell/js/composeui-node-launcher/
```

Comment:
I've removed the working directory from here.
https://github.com/morganstanley/ComposeUI/pull/563/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34

If seems that it's left here after the merge conflict resolution.
Please remove it, lerna publish has to run on the whole monorepo.